includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -204,6 +213,27 @@ public static function render_following_ui_field() {
 		<?php
 	}
 
+	/**
+	 * Render reader field.
+	 */
+	public static function render_reader_field() {
+		$value = \get_option( 'activitypub_reader_ui', '0' );
+		?>
+		<p>
+			<label>
+				<input type="checkbox" id="activitypub_reader_ui" name="activitypub_reader_ui" value="1" <?php checked( '1', $value ); ?> />
+				Enable the Reader to view posts from accounts you follow.
+			</label>
+		</p>
+		<p class="description">
+			Adds a "Social Web" page to your Dashboard where you can read posts and shares from accounts you follow. Also enables the Following feature.
```

Comment:
```suggestion
			Adds a "Social Web" interface where you can read posts and shares from accounts you follow. Also enables the Following feature. Look for it in Dashboard > Social Web.
```
Something like that to avoid ambiguity with the Dashboard itself and clarify it's in the Dashboard menu item?